### DISCUSSION TITLE
PaLM model (1.5 billion parameters) model availability in keras_nlp

### DISCUSSION THREAD

**Discussion by ramkumarkoppu** [keras-team/keras-hub#1071] [General] [answered]
Avalibility of 1.5 billion parameters PaLM model in keras_nlp

**Comment by jbischof** [comment, marked as answer]
Unfortunately this model is not open source so we cannot release it @ramkumarkoppu. You will have to use the PaLM API.

Alternatively, we are shipping an increasing number of alternative generative models like OPT. Check them out!
